tests/integration/live_data/zookeeper_tests.py

```diff
@@ -48,3 +54,23 @@ def test_create_client_with_credentials(self):
 
         secrets.get_simple.assert_called_with("secret/zk-user")
         self.assertEqual(list(client.auth_data), [("digest", "myzkuser:hunter2")])
+
+    def test_create_client_uses_threading_handler_when_not_gevent_patched(self):
+        secrets = mock.Mock(spec=SecretsStore)
+        client = zookeeper_client_from_config(
+            secrets, {"zookeeper.hosts": "%s:%d" % zookeeper_endpoint.address}
+        )
+        assert isinstance(client.handler, SequentialThreadingHandler)
+
+    @mock.patch("baseplate.lib.live_data.zookeeper.gevent_is_patched", return_value=True)
```

Comment:
🔕 You should patch this on the other test too so that your results will always be consistent when someone else messes up their tests and accidentally patches without cleaning it up.